src/main/java/com/nerdtranslator/translateapibridge/service/impl/NaturalLangApiServiceImpl.java

```diff
@@ -50,4 +41,33 @@ public String analyzeText(String textToAnalyze) {
         log.info("NaturalLangApiServiceImpl analyzeText end");
         return result;
     }
+
+    private LanguageServiceClient getLanguageService() throws IOException {
+        return LanguageServiceClient.create(
+                LanguageServiceSettings
+                        .newBuilder()
+                        .setCredentialsProvider(authenticationProvider.getCredentialsProvider())
+                        .build());
+    }
+
+    private static Document getDocument(String textToAnalyze, String textLanguage) {
+        List<String> supportedLanguages = List.of(
+                "zh", "zh-Hant", "en", "fr", "de", "it", "ja", "ko", "pt", "ru", "es");
```

Comment:
Would it be better to create any file with supported languages to dynamically load supported languages from there? I mean what if Google API add support of language not from this list?